ios/Podfile.lock

```diff
@@ -3,12 +3,11 @@ PODS:
     - AppsFlyerFramework/Main (= 6.2.4)
   - AppsFlyerFramework/Main (6.2.4)
   - Base64 (1.1.2)
-  - Batch (1.15.2)
+  - Batch (1.17.1)
```

Comment:
❓ quel est l'impact de cette montée de version ?

Reply:
le plus gros c'est qu'il faut iOS10 mtn: https://doc.batch.com/ios/sdk-changelog

Reply:
BREAKING: This version drops support for iOS 8 and 9 Batch requires Xcode 12 and iOS 10.0 or higher

Reply:
Et quelle est notre version min auj ? 
Si ça change, je préfèrerais que l'on ne monte pas de version Batch du coup

Reply:
Si c'est possible.
Sinon => mettre à jour l'OS min sur iOS et en parler aux POs

Reply:
ça build toujours bien avec l'ancienne version, je corrige